lib/sections/home.dart

```diff
@@ -1,15 +1,333 @@
+            ),
+            BottomNavigationBarItem(
+              icon: Icon(Icons.query_stats),
+              label: 'Compare',
+            ),
+            BottomNavigationBarItem(
+              icon: Icon(Icons.folder_open),
+              label: 'Reporting',
+            ),
+            BottomNavigationBarItem(
+              icon: Icon(Icons.more),
+              label: 'More',
+            ),
           ],
-        )));
+
+          // TODO: implement indices back in for the navbar
+          // currentIndex: _selectedIndex,
+          // onTap: _onItemTapped,
+        ),
+      );
+    }));
+  }
+
+  Widget build(BuildContext context) {
+    return GestureDetector(
+        onTap: () {
+          _pushInfo();
+        },
+        child: Container(
+            padding: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 8.0),
+            margin: const EdgeInsets.symmetric(horizontal: 8.0, vertical: 8.0),
+            height: 100,
+            decoration: BoxDecoration(
+                border: Border.all(
+              color: Colors.black,
+            )),
+            child: Align(
+                alignment: Alignment.center,
+                child: Row(
+                  children: [
+                    Container(
+                      child:
+                          Image.asset(_image, fit: BoxFit.fitHeight, width: 90),
+                      padding: const EdgeInsets.symmetric(
+                          horizontal: 30.0, vertical: 0.0),
+                    ),
+                    Text(_country,
+                        textScaleFactor: 1.8, textAlign: TextAlign.center)
+                  ],
+                ))));
+  }
+}
+
+// FIRST LAYER
+// List of categories (eg. population)
+class CategoryPage extends StatelessWidget {
+  // No state (like list of countries), return list of tags
+  Widget build(BuildContext context) {
+    return Container(
+        child: Center(
+            child: ListView(
+      children: [CategoryTag("Population"), CategoryTag("Mortality")],
+    )));
+  }
+}
+
+// Create tag listing a category
+// TODO: Convert to stateless if possible
+class CategoryTag extends StatefulWidget {
+  String _category = "";
+
+  CategoryTag(String category) {
+    this._category = category;
+  }
+
+  @override
+  _CategoryTagState createState() => _CategoryTagState(this._category);
+}
+
+// State for category tag
+class _CategoryTagState extends State<CategoryTag> {
+  String _category = "";
+
+  _CategoryTagState(String category) {
+    this._category = category;
+  }
+
+  // Function to push section for category
+  void _pushInfo() {
+    Navigator.of(context)
+        .push(MaterialPageRoute<void>(builder: (BuildContext context) {
+      return Scaffold(
+        appBar: AppBar(
+          title: Text(_category),
+          centerTitle: true,
+        ),
+        body: Center(
+            // Container with info on category
+            child: CategoryInfoPage()),
+        // Navbar
```

Comment:
Why is the bottom navbar being reimplemented here when it's already in main.dart?